### DISCUSSION TITLE
LXQt 2.0 with Qt6 & X11

### DISCUSSION THREAD

**Discussion by HighTechTatoka** [lxqt/lxqt#2544] [Q&A]
Hello, dear LXQt community! I wanna know, will LXQt 2.0 with Qt6 support X11? Or finally LXQt will move on the Wayland?

**Comment by stefonarch** [comment]
There was much confusion as we posted on X and mastodon that "the desktop of LXQt" is wayland-ready at 100% and this wasn't clear for many (which spread articles like "Desktop LXQt 100% wayland-ready") but it was about the desktop created by `pcmanfm-qt`.  


Wayland is still in it's early steps, with atm experimental support only for kwin but on a very good level. 
X11 support from LXQt will sure be lasting for many years, as long it is needed.

**Reply by HighTechTatoka** [reply]
Thanks for the answer. Is the Kwin best WM for LXQt for the best experience?

**Reply by stefonarch** [reply]
Depends ;)

Atm the taskbar, desktopswitcher and showdesktop looks fully working fully on Kwin, while wlroots (labwc, wayfire, sway, hyprland mostly) taskbar has still to come and will be more limited. The downside is that kwin takes some KDE dependencies in, systemsettings some more and to configure really all you need a full plasma installation.

Labwc provides a very good experience too, but atm you need a taskbar like sfwbar or waybar until our plugin is ready.

**Reply by tsujan** [reply]
I think @HighTechTatoka may mean it under X11, not Wayland. If so, KWin is also a very good WM. But it has several KDE dependencies.

**Reply by HighTechTatoka** [reply]
Ok, thanks. What WM do you use?

**Reply by tsujan** [reply]
If you asked me, I use KWIn under X11, but many LXQt users may prefer xfwm4. I'm sure there are other good WMs too.

Avoid Openbox because it's dead and buggy.

**Reply by stefonarch** [reply]
>@HighTechTatoka may mean it under X11, not Wayland. 

My bad :) But for kwin it's actually the same pro and cons on every platform...

**Reply by HighTechTatoka** [reply]
Thank you all. As I understand you both use kwin as WM of LXQt? And I do not understand something about how many people support LXQt? Only you both do that?:))

**Reply by tsujan** [reply]
> how many people support LXQt?

There are many good users, whose reports have been crucial to developing LXQt. and we have good devs and code contributors. But our manpower is small compared to the amount of the job.

**Reply by HighTechTatoka** [reply]
It is good! Does Hong Jen Yee (PCMan) still support LXQt?

**Reply by stefonarch** [reply]
It's some years that he's not active anymore on Github at all. 
His decision to migrate from Gtk to Qt was very wise.

**Comment by tsujan** [comment]
X11 will be supported as long as it exists (a very long time — no one has set a date).

LXQt is starting to have a better support for Wayland — in some places, a full support — but that doesn't mean the X11 support will be abandoned.

**Reply by HighTechTatoka** [reply]
Thanks for the answer.

**Comment by PCMan** [comment]
Hi,
I'm really lad that someone still remember me. Due to some personal reasons
it's very hard for me to keep working on this project at the moment. I'm
still a happy active user, though.
The latest LxQt is much better than what I used to have several years ago.
As a happy user I really appreciate the work done by all of you. No matter
what current active developers decide to do, you have my support. :-)

All the best,
Hong Jen-Yee

HighTechTatoka ***@***.***> 於 2024年3月19日 週二 下午12:38 寫道：

**Reply by stefonarch** [reply]
In fact we mention your name daily, with `fm-qt` suffix :)
All the best and thanks!

**Reply by tsujan** [reply]
> I'm really glad that someone still remember me

Remember? Some of us were admiring your codes a few days ago.

**Reply by HighTechTatoka** [reply]
Thank you very much for your legacy. Users, developers and WEB will not forget you.
